doc/nrf/ug_tfm.rst

```diff
@@ -97,7 +97,7 @@ TF-M employs two UART interfaces for logging: one for the secure part (MCUboot a
 The logs arrive on different COM ports on the host PC.
 
 On the nRF5340 DK, you must connect specific wires on the kit to receive secure logs on the host PC.
-Wire the pins P0.25 and P0.26 to RxD and TxD respectively.
+Wire the pins P0.25 and P0.26 to ``RxD`` and ``TxD`` respectively of the ``P24`` connector.
```

Comment:
It might not be obvious that "of the P24" connector" is talking about the RxD and TxD.

Maybe a separate sentence instead.

The RxD and TxD signals are on the P24 connector.

Reply:
RxD and TxD are written on the board.